README.md

```diff
@@ -258,6 +262,46 @@ rules:
   - name: whitelist
 ```
 
+#### Excluding Categories of Rules
+
+You can also specify any number of rule categories to be excluded, or filtered out, from within your `woke` configuration. If any rules in a configuration file have matching categories, they will be excluded and will not be run against the target files.
+
+This example will exclude `rule1` and `rule2`, but will still use `rule3`:
+
+```yaml
+exclude_categories:
+  - category1
```

Comment:
I am curious about the use case for having `exclude_categories` configured via the yaml. Why wouldn't you just remove the rules themselves? If the ability to exclude rules was needed, it would make more sense to me that you would do it via the command line (like `woke --exclude-category category1 --exclude-category category2`) so categories can be excluded dynamically without modifying the config. But if I'm deciding to modify the config to exclude categories, I don't see why I wouldn't just remove the rules themselves.

Reply:
That's a fair point! The thought behind this is mainly with potentially having configs that extend from other configs in the future. With this approach, I could have my declarative config file where I could have some control over what rules I'm importing and enabling from other rulesets. This functionality now would serve as a step towards that valuable use case (to be combined with importing / extending from remote rulesets from within a local config file, for example). Thoughts?

Reply:
That makes sense to me, will be much more valuable when remote/multiple rule sets are available to be used. Thanks!

Reply:
It seems this documentation was lost in your last merge. You'll want to add it back in https://github.com/get-woke/woke/blob/main/docs/rules.md

Reply:
You're right! Sorry about that, I added that back